dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.mpCompatible-1.0.feature

```diff
@@ -0,0 +1,10 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpCompatible-1.0
+visibility=private
+singleton=true
+Subsystem-Version: 1.0
+-features=com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0, 8.0"
```

Comment:
Since MP 1.x pulls in CDI 1.2. I think tolerating ee6 is not going to work.

Reply:
You are right that it is confusing but we did not enforce it before and we can't change that now. There are some combinations which do "work" together (start at the same time). e.g. If you only pull in mpConfig-1.1 and mpFaultTolerance-1.0 (but NOT cdi-1.2) then those should still work at the same time as (some) EE6 features because there is no actual clash.... even if it doesn't make sense!

Reply:
ok

Reply:
I had to remove the eeCompatible dependencies for now. Will try to add them back in a second PR.